content/usgs-packages/geoknife_Discovery.Rmd

```diff
@@ -0,0 +1,233 @@
+DelBay_fabric <- webdata(url="http://opendap.co-ops.nos.noaa.gov/thredds/dodsC/DBOFS/fmrc/Aggregated_7_day_DBOFS_Fields_Forecast_best.ncd")
+query(DelBay_fabric, "variables")
+
+# need to set the variable(s) in order to query the times
+variables(DelBay_fabric) <- c("Vwind", "temp")
+query(DelBay_fabric, "times")
+```
+
+Here is a second example of using a non-GDP dataset. This data was found under the [data section on Unidata's website](http://www.unidata.ucar.edu/data/#home). This is aggregated UNIWISC satellite data for Earth's "surface skin" temperature. 
+
+```{r create-fabric-UNIWISC, message=FALSE, warning=FALSE}
+skinT_fabric <- webdata(url="http://thredds.ucar.edu/thredds/dodsC/satellite/SFC-T/SUPER-NATIONAL_1km")
+skinT_var <- query(skinT_fabric, "variables")
+
+# need to set the variable(s) in order to query the times
+variables(skinT_fabric) <- skinT_var
+query(skinT_fabric, "times") # your times might be different because this is forecast data
+```
+
+Both examples we've included here use aggregated data, meaning there is a different file for each year. Some data that you encounter might not be aggregated. In these cases, you will need to create more than one geojob and join data at the end.
+
+Now that we have explored options for our webdata, let's look at what options exist for geospatial features.
+
+## Available webgeoms
+
+The next component to `geonkife` jobs is the spatial extent of the data, a.k.a. the stencil. The stencil is defined by using either of the functions `simplegeom` or `webgeom`. `simplegeom` is used to explicitly define an area by the user, but `webgeom` is used to specify an existing web feature service (WFS) as the geospatial extent. Defining your stencil using `simplegeom` will be covered in the next lesson. This lesson will just show how to learn what available webgeoms exist. Users can use any WFS url to create their stencil, but there are a number of features that exist through GDP already. To determine what features exist, you can create a default webgeom object and then query its geom name, attributes, and values. This will return all available GDP default features.
+
+```{r create-stencil, message=FALSE, warning=FALSE}
+# setup a default stencil by using webgeom and not supplying any arguments
+default_stencil <- webgeom()
+
+# now determine what geoms are available with the default
+default_geoms <- query(default_stencil, "geoms")
+length(default_geoms)
+head(default_geoms)
+```
+
+You will notice a pattern with the names of the geoms: a category followed by `:`, and then a specific name. See the table below for definitions of the categories. These category-name combinations are the strings you would use to define your geom.
+
+```{r cateogry-table, eval=TRUE, echo=FALSE, message=FALSE}
+Category <- c("upload", "sample", "draw", "derivative")
+Description <- c(
+  "upload",
+  "sample",
+  "draw",
+  "derivative"
+)
+
+library(htmlTable)
+htmlTable(data.frame(Category, Description),
```

Comment:
I don't exactly get what this table is showing.  I think it is worth pointing out that there are samples, and if you upload a shapefile to GDP it will end up in the upload workspace, but I don't know that these categories really need breakdown in this manner.